draft-ietf-quic-tls.md

```diff
@@ -328,8 +328,11 @@ encryption levels:
 
 - PADDING and PING frames MAY appear in packets of any encryption level.
 
-- CRYPTO and CONNECTION_CLOSE frames MAY appear in packets of any encryption
-  level except 0-RTT.
+- CRYPTO frames and CONNECTION_CLOSE frames signaling errors at QUIC layer (type
```

Comment:
at "the" QUIC layer (or the transport layer)

Reply:
Thanks! Applied in 408ba48.